### PR TITLE
Edit homepage and add gallery, about me, side projects, and blog posts

### PR DESCRIPTION
**Pull request by otobin**
html-outline has the bare bones for the portfolio project. It has all html pages with pictures and a baseline stylesheet, but currently does not have any javascript.

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
 
-#greeting-container {
-  margin-top: 20px;
+.main-content {
+    margin-left: 160px;
+    top: 0;
+    height: 100%;
+    position: fixed;
+    background-color: cadetblue;
+    color: #c0ccd4;
```
Comment (anicr7) [inline portfolio/src/main/webapp/style.css:25]: Not very important, but I would try to improve the contrast between the font color and background color. Here is a good way to make sure the contrast is good enough - https://contrast-ratio.com/#%23c0ccd4-on-cadetblue

File: portfolio/src/main/webapp/home.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="side-navigation">
+        <a href="home.html">Home</a>
+        <a href="about_me.html">About me</a>
+        <a href="side_projects.html">Projects</a>
+        <a href="photography.html">Gallery</a>
```
Comment (anicr7) [inline portfolio/src/main/webapp/home.html:14]: I don't see a photography.html but do see gallery.html, so fix either or the file name?

File: portfolio/src/main/webapp/blog_posts.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+           There were so many different lessons--concrete and abstract--that I took away from the course. Finally, it came to me: “The Earth is a closed system.”</p>
+      <p>During the course, my group and I had to practice Leave No Trace principles, which is exactly what it sounds like. Everything that we produced 
+          (trash, food waste, dishwater, etc…) had to be carried with us or properly disposed of. Our expedition was a closed system--there was no such 
+          thing as throwing something away. Empty food bags became drybags for headlamps and batteries, unused bandanas served as towels for dirty dishes, 
+          and we shared leftovers to prevent food waste and empty stomachs. Upon returning home, it dawned on me how often we think that throwing something 
+          away means that it disappears. Completing a NOLS course taught me a variety of easy ways that I can live my life differently in order to lessen my 
+          impact on our closed system. Here are five that particularly stood out to me: </p>
+     <p> 1. Thrift </p>
```
Comment (anicr7) [inline portfolio/src/main/webapp/blog_posts.html:33]: nit: Usually you can use <li> for ordered lists.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,14 @@
...
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const random_facts =
+      ["I'm currently learning how to juggle", "I want to set foot on all 7 continents", "My Favorite food is pesto pasta"];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const random_fact = ramdom_facts[Math.floor(Math.random() * random_facts.length)];
```
Comment (anicr7) [inline portfolio/src/main/webapp/script.js:24]: Random?  Currently the random facts button doesn't work.

File: portfolio/src/main/webapp/about_me.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="side-navigation">
+        <a href="home.html">Home</a>
+        <a href="about_me.html">About me</a>
+        <a href="side_projects.html">Projects</a>
+        <a href="photography.html">Gallery</a>
```
Comment (nsri-google) [inline portfolio/src/main/webapp/about_me.html:13]: This page does not seem to exist.

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (otobin) [reply portfolio/src/main/webapp/style.css:25]: This is a great resource, thanks for providing it!

File: portfolio/src/main/webapp/about_me.html (hunk 5 above)
Comment (otobin) [reply portfolio/src/main/webapp/about_me.html:13]: Yes, I accidentally mixed up the file names.

File: portfolio/src/main/webapp/home.html (hunk 2 above)
Comment (otobin) [reply portfolio/src/main/webapp/home.html:14]: I'll just rename the file. Thanks for pointing out the easy fix.

**Pull request merged**